man/sbd.8.pod

```diff
@@ -394,6 +394,30 @@ a fencing message by the cluster.
 
 =back
 
+=head2 query-watchdog
+
+Example usage:
+
+	sbd query-watchdog
+
+Check for available watchdog-devices and print some info.
+
+B<Warning>: This command will arm the watchdog during query and if your
+watchdog refuses disarming (nowayout-parameter set for kernel-module)
```

Comment:
comma after query

maybe "(for example, if its kernel module has the 'nowayout' parameter set)"